frontend/src/components/BalanceCard.tsx

```diff
@@ -33,7 +33,7 @@ function BalanceCard({
         <BalanceCardIcon className="h-4 w-4 text-muted-foreground" />
       </CardHeader>
       <CardContent>
-        <div className="text-2xl font-bold balance sensitive">
+        <div className="text-2xl font-bold balance sensitive slashed-zero">
```

Comment:
is `sensitive` required? we added posthog only temporarily for initial testing. It's already been removed.

Reply:
Ah, so then should I remove sensitive from other places too?

Reply:
@bumi do you think we still need this classname?

Reply:
does that hurt? 
tailwind likes classes. and actually that one is the most descriptive one of them :) 

https://imgflip.com/i/94m5ma